components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -124,6 +133,25 @@ public void testDefault(String suffix) throws FormatException, IOException {
     }
   }
 
+  @Test(dataProvider = "options")
+  public void testOptions(String options) throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tiff");
+    outFile.delete();
```

Comment:
Could we add if `File.exists{ ... }` condition over here to make sure its easier to understand the logic here. Given that this logic has been used in other tests as well, this PR can go ahead in its current state and we could handle similar cases of this logic, in a separate PR.